nilearn/_utils/data_gen.py

```diff
@@ -117,6 +122,10 @@ def generate_regions_ts(
     window : :obj:`str`, default='boxcar'
         Name of a window in scipy.signal. e.g. "hamming".
 
+    negative_regions: :obj:`bool`, default=False
```

Comment:
```suggestion
    negative_regions : :obj:`bool`, default=False
```
otherwise the html render is 'broken' (not as pretty)